ethereum/client.go

```diff
@@ -841,6 +841,43 @@ func (ec *Client) blockByNumber(
 	return r, err
 }
 
+// contractCall returns the data specified by the given contract method
+func (ec *Client) contractCall(ctx context.Context,
+	blockIndex int64, blockHash string, contractAddress string,
+	encodedData string) (map[string]interface{}, error) {
```

Comment:
```suggestion
func (ec *Client) contractCall(
        ctx context.Context,
	blockIndex int64, 
	blockHash string, 
	contractAddress string,
	encodedData string,
) (map[string]interface{}, error) {
```

Reply:
make sure to not commit using UI. it doesn't verifies the commit - so make this change in local and push it.